packages/terra-notification-dialog/docs/README.md

```diff
@@ -1,6 +1,8 @@
 # Terra NotificationDialog
 
 `terra-notification-dialog` is a notification dialog component built over the `terra-abstract-modal`. It has the highest z-index of 9001. It is a common component to be used for confirmation/acceptance criteria style dialogs.
+<br />**Note:** Use at least one of `acceptAction`, `rejectAction`, `primaryAction` or `secondaryAction` props to avoid getting a warning.
```

Comment:
> to avoid getting a warning

maybe stress the real problem that the dialog will not function as expected.

Reply:
Updated in 567c907